appcenter-data/Data.js

```diff
@@ -33,7 +35,7 @@ const Data = {
  * If the operation fails, the promise is rejected with an exception containing the details of the error.
  */
 function read(documentId, partition, readOptions) {
-    if (readOptions === undefined) {
+    if (!readOptions) {
```

Comment:
Please do not revert that as it would match 0 or false.
On typescript we will be more strict in the definition also.
null is pass through to native SDK which allows it, so it's actually better to check for undefined.

Reply:
Good point @guperrot, thanks.

Reply:
Done